app/assets/stylesheets/components/_secondary-navigation.scss

```diff
@@ -0,0 +1,94 @@
+.app-c-secondary-navigation {
+  @include govuk-font(19);
+  @include govuk-responsive-margin(6, "bottom");
+
+  display: block;
```

Comment:
Is this needed? By default, all display styling is `block` [ref](https://www.w3schools.com/css/css_display_visibility.asp#:~:text=The%20%3Cscript%3E%20element%20uses%20display:%20none;%20as%20default.)